kinto/tests/test_views_collections.py

```diff
@@ -35,12 +35,15 @@ def test_collections_name_should_be_simple(self):
                           headers=self.headers,
                           status=400)
 
-    def test_unknown_bucket_raises_404(self):
+    def test_unknown_bucket_raises_403(self):
```

Comment:
More information about this here: http://security.stackexchange.com/questions/46171/is-it-a-good-practice-to-show-403-unauthorized-access-error-to-user